avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/FeatureListGeneration.scala

```diff
@@ -160,46 +194,120 @@ class FeatureListGeneration {
    * @param params TensorizeIn parameters specified by user
    * @param fileSystem A file system
    */
-  private def writeFeatureList(params: TensorizeInParams, fileSystem: FileSystem): Unit = {
+  private def writeFeatureList(
+    params: TensorizeInParams,
+    fileSystem: FileSystem,
+    ntvTensors: mutable.HashSet[String]): Unit = {
+
+    /**
+     * Write feature list to disk for a list of (feature entry, count) pairs
+     *
+     * @param p Path to file to create and write
+     * @param featureEntriesWCount a list of (feature entry, count) pairs
+     */
+    def writeFeatureEntriesWCountToDisk(
```

Comment:
please move this method out and make it private

Reply:
Will address.